sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_digitaltwins_client.py

```diff
@@ -524,12 +525,12 @@ def delete_model(self, model_id, **kwargs):
 
     @distributed_trace
     def get_event_route(self, event_route_id, **kwargs):
-        # type: (str, **Any) -> ~azure.digitaltwins.models.EventRoute
+        # type: (str, **Any) -> DigitalTwinsEventRoute
         """Get an event route.
 
         :param str event_route_id: The Id of the event route.
-        :return: The EventRoute object.
-        :rtype: ~azure.digitaltwins.models.EventRoute
+        :return: The DigitalTwinsEventRoute object.
+        :rtype: DigitalTwinsEventRoute
```

Comment:
You will need to actually wrap the result in your new class wrapper:
```python
result = self._client.event_routes.get_by_id(event_route_id, **kwargs)
return DigitalTwinsEventRoute._from_generated(result)
```

Reply:
Not anymore. I renamed the type with a autorest transformation.